crates/pop-telemetry/README.md

```diff
@@ -0,0 +1,70 @@
+# Anonymous Usage Metrics Collection with Umami
```

Comment:
I think the title should be something simple like Telemetry with a one liner of 'Anonymous Usage Metrics Collection with Umami' within.